Swiftfin/Views/SettingsView/CustomizeViewsSettings/Components/Sections/HomeSection.swift

```diff
@@ -23,6 +30,10 @@ extension CustomizeViewsSettings {
         var body: some View {
             Section(L10n.home) {
 
+                if userSession?.user.isAdministrator ?? false {
+                    Toggle("Activity Indicator", isOn: $activeSessionIndicator)
```

Comment:
L10n

Reply:
Whoops adding that now. I'm normally pretty good about labels but I was slicing that out of https://github.com/jellyfin/Swiftfin/pull/1271 and I missed that one.